api/v1/health/server/server.go

```diff
@@ -17,13 +17,13 @@ import (
 	"strconv"
 	"sync"
 	"sync/atomic"
-	"syscall"
 	"time"
 
 	"github.com/go-openapi/runtime/flagext"
 	"github.com/go-openapi/swag"
 	flags "github.com/jessevdk/go-flags"
 	"golang.org/x/net/netutil"
+	"golang.org/x/sys/unix"
```

Comment:
This file is autogenerated, so this change will be reverted the next time it's generated. You'd probably need to change `go-swagger` to generate `x/sys/unix` instead of `syscall`.

Reply:
Good catch, resolving this has opened up a bit of a yak shaving thread.

Effectively cilium:master has stale swagger generated sever code, and which is hoping to be rectified by: https://github.com/cilium/cilium/pull/10336

Reply:
Thanks for taking care of it!

Reply:
I've removed this change to the generated file. `/api/v1/server.gotmpl` has been updated to use the `unix` package. However there are breaking changes when running `swagger` at the moment. I [outlined](https://github.com/cilium/cilium/pull/10336#issuecomment-591107088) the issues in #10336, once those are resolved, regenerating the APIs should pick up the unix package changes.